scenarios/reproducers/3-nodes.yml

```diff
@@ -0,0 +1,66 @@
+---
+cifmw_install_yamls_repo: /home/cjeanner/work/github/openstack-k8s-operators/install_yamls
```

Comment:
I actually do not mind having a specific dir in the $HOME as documented recommendation.
Probably more specific like 'cifmw' instead of work .
Might be put here by var.

Reply:
maybe just commenting out and stating it an example is good.

Reply:
ah, right, that's a missed thing with my own env, I forgot to update it before pushing.